backend/app/models/campaign.rb

```diff
@@ -26,6 +26,23 @@ class Campaign < ApplicationRecord
                             message: 'is mot of a supported file type. Please upload a PNG, JPG or JPEG file.' },
             size: { less_than: 1.megabytes, message: 'must be less than 1MB.' }
 
+  def donation_breakdown
```

Comment:
Can we add a validation to allow only MAX 5 charities per campaign? 

Sorry can't add the comment at the relevant line.